### PR TITLE
Qa fixes

### PR DESCRIPTION
**Pull request by talhainvenxion**
https://zenml.atlassian.net/browse/DAS-411
https://zenml.atlassian.net/browse/DAS-348
https://zenml.atlassian.net/browse/DAS-308
https://zenml.atlassian.net/browse/DAS-291

### CODE REVIEW

File: src/ui/layouts/pipelines/RunsTable/HeaderCols/index.tsx [hunk 1]
```diff
@@ -364,7 +364,7 @@ export const useHeaderCols = ({
                 color="black"
                 style={{ fontSize: '14px' }}
               >
-                PIPELINE
+                PIPELINEasdasd
```
Comment (Cahllagerfeld) [inline src/ui/layouts/pipelines/RunsTable/HeaderCols/index.tsx:367]: This shouldn't be committed I think 🤔

**Review by Cahllagerfeld — COMMENTED** [review]
To me the changes for `DAS-348` and `DAS-308` are looking good 👍 
I'm not sure how to reproduce `DAS-411`, but from the code perspective I think this is fine. Did we catch all occurences of this bug here, or do you know how to reproduce it?

For `DAS-291` is was testing the secrets dropdown, where Arrow Up and down works, entering the selected value with `Tab` didn't work though. Did I test in the right place?

File: src/ui/layouts/pipelines/RunsTable/HeaderCols/index.tsx (hunk 1 above)
Comment (Talha1995) [reply src/ui/layouts/pipelines/RunsTable/HeaderCols/index.tsx:367]: Oops, It was a mistake.

**Comment by Talha1995** [comment]
> To me the changes for `DAS-348` and `DAS-308` are looking good 👍 I'm not sure how to reproduce `DAS-411`, but from the code perspective I think this is fine. Did we catch all occurences of this bug here, or do you know how to reproduce it?
> 
> For `DAS-291` is was testing the secrets dropdown, where Arrow Up and down works, entering the selected value with `Tab` didn't work though. Did I test in the right place?

for `DAD-411` you can reproduce it on https://demo.zenml.io/workspaces/default/all-runs/132b1b0b-01ee-4314-af24-028058c4d1a1/dag I tested it and fixed for all cases.
for 'DAS-291' yes you are testing it in the right place, selection works whenever the user press 'enter'. Do we need to perform the same action 'Tab' also?

**Pull request merged**